tests/cbmc/proofs/s2n_array_free/s2n_array_free_harness.c

```diff
@@ -22,10 +22,22 @@ void s2n_array_free_harness()
 {
     /* Non-deterministic inputs. */
     struct s2n_array *array = cbmc_allocate_s2n_array();
-    __CPROVER_assume(s2n_result_is_ok(s2n_array_validate(array)));
 
+    /* Assumptions. */
     nondet_s2n_mem_init();
+    __CPROVER_assume(s2n_result_is_ok(s2n_array_validate(array)));
 
     /* Operation under verification. */
-    s2n_array_free(array);
+    s2n_result result = s2n_array_free(array);
+    if (s2n_result_is_error(result)) {
+        assert(s2n_errno != S2N_ERR_FREE_STATIC_BLOB);
+    }
+
+    /* Cleanup after expected error cases, for memory leak check.
+       It's good proof practice not to mix state mutations (below) with property checks (above). */
```

Comment:
Please audit your comments for S2N style

Reply:
Thanks, I keep forgetting! I wish there was a CI pass for checking this.